cli/src/commands/describe.rs

```diff
@@ -151,8 +163,12 @@ pub(crate) fn cmd_describe(
             .map(|commit| -> Result<_, CommandError> {
                 let mut commit_builder = tx.repo_mut().rewrite_commit(commit).detach();
                 if commit_builder.description().is_empty() {
-                    commit_builder
-                        .set_description(tx.settings().get_string("ui.default-description")?);
+                    let description = if let Some(description) = &shared_description {
+                        description.into()
+                    } else {
+                        tx.settings().get_string("ui.default-description")?
+                    };
+                    commit_builder.set_description(description);
```

Comment:
```suggestion
                    commit_builder.set_description(match shared_description {
                        Some(description) => description.to_owned(),
                        None => tx.settings().get_string("ui.default-description")?,
                    });
```

suggestion: I recommend not using `into` since it's not obvious (without IDE support) what type conversion is happening, and it can result in unexpected behavior in future refactorings. If this is meant to convert from `&str` to `String`, then maybe `.to_owned()` is best here.

nit: My preferred way of writing this might be via an inline `match`, but not a big deal if you prefer an explicit variable and `if`.